simiyu1/JavaDevsNairobi
# 156005300 fastlane deploy to slack

#### What does this PR do?
 - Integrate fastlane deployment to slack

#### Description of Task to be completed?
 - call slack webhooks from fastfile
 - create lane for slack in fastfile
 
#### How should this be manually tested?
 - Clone this repository to your machine, select app and run the application on an emulator or device.
 - on your terminal run nano ~/.zshrc to edit the file
 -  then in the file write 
```
     export LC_ALL=en_US.UTF-8
     export LANG=en_US.UTF-8
     export SLACK_URL="...Replace with workspaceURL..."
     export SLACK_API_TOKEN="...Replace with actual token..."
     export SLACK_CHANNEL="...Replace with actual channel..."
```
- Save and close nano and then run command `bundle exec fastlane slackbuild` 
- It should generate an APK into the slack channel indicated above.

#### What are the relevant pivotal tracker stories?
 - [#156005300](https://www.pivotaltracker.com/story/show/156005300)

#### Screenshots
![image](https://user-images.githubusercontent.com/17185082/46878503-28137400-ce4c-11e8-8264-172c39ed2acf.png)
